From 454a8e2bda07ecceaa2b0114d5fd01b16028fcc0 Mon Sep 17 00:00:00 2001
From: Andreas Dangel <andreas.dangel@pmd-code.org>
Date: Thu, 5 Sep 2024 14:57:32 +0200
Subject: [PATCH] test case 4 - unrelated change

This commit changes a file in /docs. This definitely doesn't
change any code, so the regression tester should skip the
execution.
---
 docs/index.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index 3bafc43f4d..ac7b593dfd 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -17,7 +17,7 @@ additional_js:
  - js/shuffle.js
 ---
 
-
+<!-- some change in docs -->
 
 ## ðŸ’¡ Overview
 
-- 
2.45.2

